static/src/js/components/SearchPanels/SearchPanels.jsx

```diff
@@ -180,10 +180,18 @@ class SearchPanels extends PureComponent {
 
     const {
       pageNum: collectionsPageNum = 1,
-      sortKey: collectionsSortKey = collectionSortKeys.scoreDescending
+      paramCollectionSortKey: urlCollectionsSortKey
     } = collectionQuery
 
-    const [activeCollectionsSortKey = collectionSortKeys.scoreDescending] = collectionsSortKey
+    const { collectionSort: userPrefCollectionSortKey } = preferences
```

Comment:
Since the test for this component is `static/src/js/components/SearchPanels/__tests__/SearchPanels.test.jsx` already just passing in state `collectionQuery` we having things passing since we are maintaining the default behavior but, we should add a test that is passing this new one into the component